README.md

```diff
@@ -1,15 +1,15 @@
 
 # Implementation Details
 
-In this project we simulate the integration with a live Ad Server (CSAI use case) or SSAI provider through a mock ad playlist configuration. This is meant to capture the stream's ad pods, including their duration and the reference to the true[X] payloads in each pod. This configuration is maintained in `res/reference-app-streams.json` as part of the `vmap` key. In this sample channel, two ad breaks are defined, `preroll` and `midroll-1`. The duration of the true[X] specific Choice Card video asset is called out as well as the duration of the standard video ads. This is a simplified representation of what would otherwise come through a provider-dependent XML or JSON syntax, but should be sufficient to exemplify the flow. The stream location itself is maintained in the `url` value.
+For the purpose of this project, some of the data is hard coded as the primary goal here is to showcase the RAF - TrueX Ad integration for client side handled ads. The core of this implementation starts in `PlaybackTask`. As a brief summary, the app will launch and prepare the TrueX Ad Renderer (TAR). Once ready, the app will launch to a simple page (`DetailsFlow`), and waits for the user to start playback by selecting the one/play action. Upon selecting the action, the `RafContentFlow` is started, which sets up some UI elements, and then invokes `PlaybackTask` where most of the action starts.
 
-This `vmap` ad playlist is marshaled through to the `ContentFlow` SceneGraph Component which handles the stream playback. In `preprocessVmapData` we parse out this ad playlist and build simple data structures which are then referenced as part of the video position change handler (`onVideoPositionChange`) to detect when we encounter a true[X] ad pod. We then initialize and launch the true[X] ad with the `isOneStageIntegration` flag, which indicates to the TruexAdRenderer that it should defer to the host channel's video player and stream for choice card asset rendering. Note that because we defer we also need the channel to notify the TruexAdRenderer when playback has moved on past the choice card, which we do so also from `onVideoPositionChange` using the `stop` action.
+In the task, we initialize the player with video metadata and RAF with ads. A very basic use case, wrapped in `vmap-truex.xml` is the default ad payload, with a preroll and midroll, each containing a TrueX ad. The payload is meant to be a simulation of a valid ad payload from the ad server. While the preroll and midroll are also local files, the TrueX ads are real server side ads, and can be seen in `truex-pod-<preroll/midroll>`. The assumption here is the ad payload is able to be parsed by RAF via `RAF.setAdUrl(url)`. RAF is fairly flexible but it is a black box. In general as long as the ad payload is a standard format, it will parse correctly. Once RAF has been setup, playback is ready to start.
```

Comment:
I would reword a bit to make it clearer that the `vmap-truex.xml` is a proxy / mock for a live ad server response and a real world app would simply substitute in its ad server URL / tag. RAF does the rest (abstracting away the details / specifics of the ad server being used) and the rest of the RAF flow should work as demonstrated.